snuba/cli/consumer.py

```diff
@@ -179,6 +180,10 @@ def consumer(
 
     setup_logging(log_level)
     setup_sentry()
+
+    logger.info("Checking Clickhouse connections")
+    check_clickhouse_connections()
```

Comment:
Consumers generally only connect to 1 cluster, mind just checking the one instead of the whole lot?

Reply:
ok. update the PR to allow passing in cluster as a parameter